learn/core_concepts/relevancy.md

```diff
@@ -194,3 +194,7 @@ For a more detailed look at this subject, see our reference page for [the search
 ```
 
 With the above attribute ranking order, matching words found in the `title` field would have a higher impact on relevancy than the same words found in `description` or `release_date`. If you searched "1984", for example, results like Michael Radford's film "1984" would be ranked higher than movies released in the year 1984.
+
+:::note
+Please keep in mind that **the order of the results depends on the position of the `attribute` rule in the [`rankingRules`](#default-order) object.** Meaning, **if `attribute` is at the bottom of the ranking rules list, it will have no impact.**
```

Comment:
In my opinion, too much emphasis is almost the same as no emphasis. Like, it should be something that tells the reader what is the most important thing about a specific section of text, so if we say everything's important, well, we're not helping the reader find the crucial stuff more quickly.

Also suggesting a few minor wording changes, but they're not super important.

```suggestion
The order of the results depends on the position of the `attribute` rule in the [`rankingRules`](#default-order) array. Meaning, **if `attribute` is at the bottom of the ranking rules list, it will have almost no impact on your search results.**
```

Reply:
I think this wording change is important, since it will have an impact on your search results, just a very minor one (tie-breaking documents that were equally relevant under all the previous ranking rules, as I understand it).